app/src/main/res/drawable/ic_edit_18.xml

```diff
@@ -1,6 +1,7 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
   android:width="18dp"
   android:height="18dp"
+  android:autoMirrored="true"
```

Comment:
This should not be reversed. Only icons which represent direction should be reversed.

Reply:
Done.